libs/csm/rest/csm_rest_quota.py

```diff
@@ -252,8 +252,8 @@ def verify_user_capacity(self, user_id, used:int, used_rounded:int=None, obj_cnt
             self.log.info("Expected: %s", user_id)
 
             if user_id != auser_id:
-                self.log.info("User ID mismatch")
-                result = False
+                self.log.error("User ID mismatch")
+                #result = False
```

Comment:
Can u uncomment this?
We dont want false positive on main.

Reply:
done